### DISCUSSION TITLE
Running bundle config

### DISCUSSION THREAD

**Discussion by mountriv99** [ruby/setup-ruby#528] [General]
Hey guys,

I need to run `bundle config build.nokogiri --use-system-libraries` before running bundle install in order to install nokogiri, but I couldnt figure out how/where to set this config.

Or maybe there's another (proper) way that allows me to install nokogiri?

Appreciate if someone can help me out, thanks

**Comment by eregon** [comment]
https://github.com/ruby/setup-ruby#bundle-config

**Reply by mountriv99** [reply]
Thanks for you answer!
I have been trying this out but without much luck... I'm still kind of confused what to do here. Do I just add a ./bundle/config file in the root directory and add this line into the file:
`bundle config build.nokogiri --use-system-libraries`?

**Comment by mountriv99** [comment]
problem went away after i upgraded nokogiri to 1.15, closing thread
